standards-catalogue/.schema/standards.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://github.com/alphagov/data-standards-authority/blob/main/standards-catalogue/.schema/standards.json",
+  "title": "Data Standards Authority Standards objects",
+  "description": "A JSON Schema to manage the format of Standards objects",
+  "type": "object",
+  "required": [
+    "name",
+    "contraction",
+    "specification_url",
+    "maintainer_id",
+    "endorsement_status_events",
+    "tags",
+    "license_id"
+  ],
+  "properties": {
+    "identifier": {
+      "$comment": "The unique identifier assigned to this standard",
+      "type": "string"
+    },
+    "name": {
+      "$comment": "The title or name of the standard",
+      "type": "string",
+      "maxLength": 70
+    },
+    "contraction": {
```

Comment:
is it likely there'll be standards that don't have a shortened name? Wondering what people would do here if there wasn't one

Reply:
For instance, `360 Giving Standard` became `360GS` so maybe in these cases it could just be an acronym?

Reply:
fair enough